daterangepicker.js

```diff
@@ -521,6 +519,18 @@
             this.updateMonthsInView();
         },
 
+        setSingleDatePicker: function(is_single_date) {
```

Comment:
This method is exposed externally, and is used by the `search-form-jquery-client` to change the trip type